rtv/terminal.py

```diff
@@ -145,6 +155,40 @@ def suspend():
         finally:
             curses.doupdate()
 
+    @staticmethod
+    def select_program(file_url):
+        """
+        If there is a valid program to open the link, will return it, otherwise
+        returns None and the link will be opened with browser.
+        """
+        mimetype = mimetypes.guess_type(file_url)[0]
+        d = mailcap.getcaps()
+        try:
+            m = mailcap.findmatch(d, mimetype)
```

Comment:
``m`` is not very descriptive, would be more clear if it was something like ``cmd, entry = mailcap.findmatch(d, mimetype)``.